Source/monster.cpp

```diff
@@ -1067,24 +1067,19 @@ void MonsterHitMonster(Monster &monster, int i, int dam)
 	HitMonster(monster, dam);
 }
 
-void StartDeathFromMonster(int i, int mid)
+void StartDeathFromMonster(Monster &killerMonster, Monster &killedMonster)

```

Comment:
having variables names which differ by only a single letter is very confusing, `attacker` and `target` like other functions will be fine, we already know they're `Monster` instances due to the variable type.

Reply:
Thanks, attacker and target are much better names indeed 😄